### PR TITLE
remove `CommonOptions.get_value`

### PR DESCRIPTION
**Pull request by beauxq**
avoid using strings to get option values

### CODE REVIEW

File: worlds/stardew_valley/test/checks/option_checks.py [hunk 1]
```diff
@@ -49,15 +31,16 @@ def assert_cannot_reach_island(tester: SVTestBase, multiworld: MultiWorld):
 
 
 def assert_can_reach_island_if_should(tester: SVTestBase, multiworld: MultiWorld):
-    include_island = is_setting(multiworld, options.ExcludeGingerIsland.internal_name, options.ExcludeGingerIsland.option_false)
+    sd_options = get_stardew_options(multiworld)
```
Comment (agilbert1412) [inline worlds/stardew_valley/test/checks/option_checks.py]: I hate the acronym "sd" with a passion. I don't like using acronyms in code as a whole, so I'd just write "options" or "stardew_options", but if you really want to use an acronym, "sv" would be the proper one.

File: worlds/stardew_valley/test/checks/option_checks.py [hunk 2]
```diff
@@ -80,11 +63,10 @@ def assert_has_deluxe_scarecrow_recipe(tester: SVTestBase, multiworld: MultiWorl
 
 
 def assert_festivals_give_access_to_deluxe_scarecrow(tester: SVTestBase, multiworld: MultiWorld):
-    has_festivals = is_not_setting(multiworld, options.FestivalLocations.internal_name, options.FestivalLocations.option_disabled)
+    sd_options = get_stardew_options(multiworld)
+    has_festivals = sd_options.festival_locations.value == options.FestivalLocations.option_disabled
```
Comment (agilbert1412) [inline worlds/stardew_valley/test/checks/option_checks.py]: This is an error. The condition was for **not** being disabled, as in, having any festival setting. But the new condition is for being disabled instead.

It should be "!=" instead

File: worlds/stardew_valley/test/checks/option_checks.py [hunk 3]
```diff
@@ -80,11 +63,10 @@ def assert_has_deluxe_scarecrow_recipe(tester: SVTestBase, multiworld: MultiWorl
 
 
 def assert_festivals_give_access_to_deluxe_scarecrow(tester: SVTestBase, multiworld: MultiWorld):
-    has_festivals = is_not_setting(multiworld, options.FestivalLocations.internal_name, options.FestivalLocations.option_disabled)
+    sd_options = get_stardew_options(multiworld)
```
Comment (agilbert1412) [inline worlds/stardew_valley/test/checks/option_checks.py]: acronym comment again

**Review by agilbert1412 — CHANGES_REQUESTED** [review]

**Review by agilbert1412 — APPROVED** [review]

**Comment by beauxq** [comment]
@alwaysintreble in case you didn't see that [agilbert1412](https://github.com/agilbert1412) approved this

**Pull request merged**